engine/src/deposit_event.rs

```diff
@@ -258,12 +258,13 @@ impl DepositedEvent {
     pub fn from_log_entry_data(data: &[u8]) -> Result<Self, error::ParseError> {
         let event = EthEvent::fetch_log_entry_data(DEPOSITED_EVENT, Self::event_params(), data)
             .map_err(error::ParseError::LogParseFailed)?;
-        let sender = event.log.params[0]
+        let raw_sender = event.log.params[0]
             .value
             .clone()
             .into_address()
             .ok_or(error::ParseError::InvalidSender)?
             .0;
+        let sender = Address::from_slice(&raw_sender);
```

Comment:
Could probably write as `Address::new(raw_sender)` because I think `into_address()` above would return `H160`.

Reply:
It can't